src/edit/DistortableImage.EditToolbar.js

```diff
@@ -101,13 +101,98 @@ var EditOverlayAction = LeafletToolbar.ToolbarAction.extend({
 				title: 'Export Image'
 			}
 		},
-		
+
 		addHooks: function ()
 		{
 			var editing = this._overlay.editing;
 
 			editing._toggleExport();
-			this.disable(); 
+			this.disable();
+		}
+	}),
+
+	EnableEXIF = EditOverlayAction.extend({
```

Comment:
Ooh wow!!! 

Do you think we should start moving these into separate files, though, to keep the length of this file a bit lower? 

This is great work!!!

Reply:
@jywarren I will mark this draft PR for review after some refactoring changes (including separating files) as I mentioned in the initial commit above. I had this (and some other changes) in mind but since I was working on multiple PRs back then so I just drafted this one here and tended to the others meanwhile. Sorry for any inconvenience/confusion caused! I'll push the changes very soon!